### ISSUE TITLE
Boost usefulness of created OSM notes: Let users attach a photo

### ISSUE THREAD

**Issue by westnordost** [mapsme/omim#14167]
## The idea

The usefulnuss of notes created by Maps.Me is limited, because it often/usually requires an independent survey to confirm it, especially in cases of "place does not exist anymore or never existed". So, notes created by Maps.Me often linger for years and years without ever being actually resolved.

If the user attached a photo to it, such a note would be immediately actionable by an "armchair mapper". Whoever processes the note could maybe then even add more detail, for example add what (POI) is there instead.

## Implementation

OSM does not allow to attach photos to a note, but you could do what StreetComplete does: Upload the photo somewhere else and link to it! Note review tools like [NotesReview](https://ent8r.github.io/NotesReview/) even show the linked to picture directly in their UI as if it has really been attached to the note.

![image](https://user-images.githubusercontent.com/4661658/120074410-07e86000-c09d-11eb-9550-159818086fbc.png)

### But where to upload the photo to?

Using a third party (imgur etc) for that is always problematic, because such usage may be a breach of their terms and conditions. Also, there is no good control over the lifetime of the photo - ideally it could be removed when the note is closed, but it shouldn't be removed before (or much after).

Self-hosting a simple image-upload on the other hand comes with some problems like the necessity to possibly remove it due to copyright claims or otherwise.

Again, you could do what StreetComplete does. The backend for the photo-upload service is here and anyone even with just a shitty webspace that runs PHP+MySQL could set it up in 5 minutes: https://github.com/streetcomplete/sc-photo-service
In a nutshell, how it works in StreetComplete:
- photos are uploaded into private folder and are activated (=moved to public folder) by referring to the photo in an OSM note. When the note is closed, the photo is also deleted (by a cron job)
- StreetComplete only allows uploading photos (so, not any image from the photo gallery), this is a soft measure to prevent abuse  or copyrighted material to be uploaded

**Comment by westnordost** [comment]
Some stats from my photos hosting for StreetComplete:
- hosting images for 2.5 years now
- 43000 photos were uploaded in that time
- 77% have been deleted again (because their associated notes have been solved)
- webspace used by photos currently: 2.4 GB (photos are scaled down to WXGA by the app)

**Comment by matkoniecz** [comment]
See #14157 - it appears that this repository is defunct and no longer used
